### ISSUE TITLE
SHould https be used for servers?

### ISSUE THREAD

**Issue by SimonCropp** [dotnet/symstore#358]
At the moment http is used for symbol servers `http://msdl.microsoft.com/download/symbols/`. Should this be changed to https?

**Comment by mikem8361** [comment]
Yes, we should be using https.  Fixed in PR: https://github.com/dotnet/symstore/pull/359
